front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/BaseCarrousel/ScrollableCarrousel/index.tsx

```diff
@@ -92,7 +90,13 @@ const ScrollableCarrousel = ({
   }, [scrollContainerRef, hasMore, handleButtonVisiblity]);
 
   return (
-    <>
+    <Box
+      /* We set position relative to be able to position the skip (& scroll?) buttons.
+        I'd actually prefer to move the SkipButton's absolute positioning styles onto a Box that wraps the button here, so the relative/absolute relationship is clearer.
+        Similarly, I think the ScrollButtons could go inside HorizontalScroll so the positioning relationship could be made clearer.
+      */
```

Comment:
I think it's totally fine like this. I would say either make those changes, or keep it as it is, but if it's working well I wouldn't add this very long comment

Reply:
Sure, I'll remove it